jsdojo/RESTAPI_task/javascriptAPI.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+    'use strict';
+    kintone.events.on('app.record.create.show', (event) => {
+        //  get form settings
+        const params = {
+            'app': 41
+        };
+        kintone.api(kintone.api.url('/k/v1/app/form/fields', true), 'GET', params, (forms) => {
+            let newRow;
+            const setRecord = kintone.app.record.get();
+            const action = forms.properties.table.fields.action_5.options;
+            for (const actions in action) {
+                newRow = {
```

Comment:
You should be able to create the `newRow` variable with `const` here instead of creating one outside the `for...in` scope.